components/dashboard/src/menu/Menu.tsx

```diff
@@ -99,7 +99,7 @@ export default function Menu() {
                             </ul>
                         </nav>
                         {/* Hide normal user menu on small screens */}
-                        <UserMenu user={user} className="hidden md:block" />
+                        <UserMenu user={user} className="hidden md:block h-10" />
```

Comment:
**thought**: For some reason, this was needed to not break the layout around the avatar . 🤷